core/trino-main/src/main/java/io/trino/execution/executor/dedicated/ThreadPerDriverTaskExecutor.java

```diff
@@ -63,23 +67,30 @@ public class ThreadPerDriverTaskExecutor
     @GuardedBy("this")
     private boolean closed;
 
+    @GuardedBy("this")
+    private int runningLeafDrivers;
+
     @Inject
     public ThreadPerDriverTaskExecutor(TaskManagerConfig config, Tracer tracer, VersionEmbedder versionEmbedder)
     {
         this(
                 tracer,
                 versionEmbedder,
                 new FairScheduler(config.getMaxWorkerThreads(), "SplitRunner-%d", Ticker.systemTicker()),
-                config.getMaxDriversPerTask());
+                config.getMinDriversPerTask(),
+                config.getMaxDriversPerTask(),
+                config.getMinDrivers());
```

Comment:
this (minDrivers) gets passed to `targetGlobalLeafDrivers` parameter.
the name difference is significant. I think we should align the names.

Reply:
Target leaf drivers more accurately represents what it does. I didn't want to change it (yet) since it's shared across both scheduler implementations.

Reply:
if we cannot fix the config name yet, maybe we can have a warning/TODO comment there explaining the name can mislead the reader?